app/assets/stylesheets/govuk-overrides.scss

```diff
@@ -49,3 +49,22 @@
     }
   }
 }
+
+// ______________________________________________________
+
+// Search
+
+.apply-sort {
+  float: right;
```

Comment:
This doesn't belong in this file, which is originally intended for modifications to the standard GOV.UK style. This rather belongs to components.scss